modyn/selector/internal/grpc/selector_grpc_servicer.py

```diff
@@ -49,9 +51,50 @@ def __init__(self, selector_manager: SelectorManager, sample_batch_size: int):
 
     def register_pipeline(self, request: RegisterPipelineRequest, context: grpc.ServicerContext) -> PipelineResponse:
         logger.info(f"Registering pipeline with request - {str(request)}")
-        pipeline_id = self.selector_manager.register_pipeline(request.num_workers, request.selection_strategy.value)
+
```

Comment:
Hmm, it's actually not optimal that the selector needs to know all of this but this is a result of the selector registering the pipeline. Can you create an issue that states we should rework pipeline registration to the supervisor service, when we change to the supervisor service model? thank you!

Reply:
Sure! Done in #302.

Reply:
Done